src/lib/filters/scraps/scrapPing.go

```diff
@@ -36,17 +37,42 @@ func CLIPingScrap(s *string) (a *utils.TypePingScrap) {
 // CLIFLoodPingScrap returns packet loss
 func CLIFLoodPingScrap(s *string) (a *utils.TypeFloodPingScrap) {
 	arr := strings.Split(*s, "\n")
-	sentence := arr[3]
-	words := strings.Split(sentence, ", ")
+	// Packetloss line is of the form:
+	// "10 packets transmitted, 2 packets received, 80.0% packet loss"
+	var packetLossLine string = ""
+	// PingValues line is of the form:
+	// round-trip min/avg/max/stddev = 91.299/96.281/101.264/4.983 ms (Mac OS/Unix like systems)
+	// rtt min/avg/max/stddev = 91.299/96.281/101.264/4.983 ms (Linux)
+	var pingValuesLine string = ""
+	// The results for flood ping outputs
+	// are different across different platforms.
+	// So, it was not wise enough to declare an
+	// array index, but rather find the line of occurence.
+	for _, line := range arr {
+		if strings.Contains(line, "packet loss") {
+			packetLossLine = line
+		} else if strings.Contains(line, "round-trip") {
+			pingValuesLine = line
+		} else if strings.Contains(line, "rtt") {
+			pingValuesLine = line
+		}
+	}
+	// This error handler is for an optional security
+	// if we could not find suitable parameters in the
+	// ping command's output.
+	if packetLossLine == "" || pingValuesLine == "" {
+		panic("Couldn't find a suitable response which contains packet loss")
```

Comment:
Do we really need to stop bench-routes here? If not, please consider using `logger.Terminal("p", "couldn't find a suitable response which contains packet loss")`.

Also please note that error lines should not contain any capital letters.